### PR TITLE
Started Thesis section

### PR DESCRIPTION
**Pull request by guillermogb**
@supitalp ptal

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -556,6 +557,59 @@ IEEE Int. Conf. Robotics and Automation (ICRA), Hong Kong, 2014, pp. 2862-2867.
...
+*[Event-Based Feature Detection, Recognition and Classification](https://tel.archives-ouvertes.fr/tel-01426001)*,  
+Ph.D. Thesis, Université de Pierre et Marie Curie, Paris, France, 2016. [PDF](https://tel.archives-ouvertes.fr/tel-01426001/document)
+- Li, C.,  
+*[Two-stream vision sensors](https://doi.org/10.3929/ethz-b-000164862)*,  
+Ph.D. Thesis, ETH-Zurich, Zurich, Switzerland, 2017.
+- Neil, D.,  
+*[Deep Neural Networks and Hardware Systems for Event-driven Data](https://doi.org/10.3929/ethz-b-000168865)*,  
+Ph.D. Thesis, ETH-Zurich, Zurich, Switzerland, 2017. [PDF](https://www.research-collection.ethz.ch/bitstream/handle/20.500.11850/168865/phd_thesis.pdf?sequence=3&isAllowed=y)
+- Mueggler, E.,  
+*[Event-based Vision for High-Speed Robotics]()*,
```
Comment (eliasm) [inline README.md]: Link: http://rpg.ifi.uzh.ch/docs/PhD17_Mueggler.pdf

File: README.md [hunk 2]
```diff
@@ -556,6 +557,59 @@ IEEE Int. Conf. Robotics and Automation (ICRA), Hong Kong, 2014, pp. 2862-2867.
...
+Ph.D. Thesis, Université de Pierre et Marie Curie, Paris, France, 2016. [PDF](https://tel.archives-ouvertes.fr/tel-01426001/document)
+- Li, C.,  
+*[Two-stream vision sensors](https://doi.org/10.3929/ethz-b-000164862)*,  
+Ph.D. Thesis, ETH-Zurich, Zurich, Switzerland, 2017.
+- Neil, D.,  
+*[Deep Neural Networks and Hardware Systems for Event-driven Data](https://doi.org/10.3929/ethz-b-000168865)*,  
+Ph.D. Thesis, ETH-Zurich, Zurich, Switzerland, 2017. [PDF](https://www.research-collection.ethz.ch/bitstream/handle/20.500.11850/168865/phd_thesis.pdf?sequence=3&isAllowed=y)
+- Mueggler, E.,  
+*[Event-based Vision for High-Speed Robotics]()*,  
+Ph.D. Thesis, University of Zurich, Zurich, Switzerland, 20917.
```
Comment (eliasm) [inline README.md]: 20917 -> 2017 :sweat_smile:

Comment (guillermogb) [inline README.md]: Thanks. You may merge the PR

Comment (guillermogb) [inline README.md]: Once I edit it

**Pull request merged**